package.json

```diff
@@ -127,9 +127,9 @@
     "postcss-url": "^10.1.1",
     "prettier": "2.2.1",
     "pretty-quick": "3.1.1",
-    "react": "16.14.0",
+    "react": "^17.0.2",
```

Comment:
Shouldn't we update `@types/react` as well? I see that we have in a `resolutions` https://github.com/contentful/forma-36/blob/79e6290387e55fe25bd5aef1eff226b2b989d84e/package.json#L45-L46

Reply:
done